client-java/sql/src/main/java/org/evomaster/client/java/sql/internal/SqlSelectBuilder.java

```diff
@@ -0,0 +1,50 @@
+package org.evomaster.client.java.sql.internal;
+
+import org.evomaster.client.java.controller.api.dto.database.schema.DatabaseType;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SqlSelectBuilder {
+
+    public static String buildSelect(
+            DatabaseType databaseType,
+            SqlTableId tableId,
+            List<SqlColumnId> columnIds
+    ) {
+        if (columnIds == null || columnIds.isEmpty()) {
+            throw new IllegalArgumentException("Column list cannot be empty");
+        }
+
+        String columnList = columnIds.stream()
+                .map(SqlColumnId::getColumnId)
+                .collect(Collectors.joining(", "));
+
+        return String.format("SELECT %s FROM %s", columnList, buildQualifiedTableName(databaseType, tableId));
+    }
+
+    private static String buildQualifiedTableName(DatabaseType databaseType, SqlTableId tableId) {
```

Comment:
i haven't double-check the whole EM code, but i m quite sure a functionality like this would be used in different places. might already being there somewhere. instead of being a private static method here, could be directly in SqlTableId, as it would be used in more places, isn't it?